_data/packages.yml

```diff
@@ -426,7 +426,30 @@
 
     export const SFOTH = getMap("SFOTH");
     export const City = getMap("City");
-    
+
     // Then you can easily import the needed resources into another library
     import { Chatted, Cleanup } from "ReplicatedStorage/RemoteEventDefinitions";
     ```
+
+- name: rbx-bitbuffer
+  author: BinaryProvider
+  keywords: data store binary storage compression
+  tabs: demo
+  description: |-
+    The BitBuffer module is a module that was created by Stravant that allows for the reading and manipulation of binary based data storage, which is an essential module if you have a lot of data you need to store compactly.
+
+    ***
+
+    ```ts
+    import BitBuffer from "rbx-bitbuffer";
+    // Save data
+    const buffer = BitBuffer.create();
+    buffer.writeString('Example data');
+    buffer.writeBool(false);
+    const data = buffer:toBase64();
+    // Load data
+    const buffer = BitBuffer.create();
+    buffer:fromBase64(data);
```

Comment:
This isn't valid TS. Oops?

Reply:
Sorry, it's not clear enough :)
It's not meant to be used in the same scope. I'll update to make it clearer.

Thanks for the feedback!

Reply:
Actually, I meant that `buffer:toBase64();` isn't valid TS. I think you meant `buffer.toBase64();`

Reply:
Omg.. how could I miss that. Copy Paste typo, sorry about that. It should be corrected now.